packages/ui-popover/src/Popover/index.tsx

```diff
@@ -86,72 +89,50 @@ class Popover extends Component<PopoverProps> {
-    onMouseOut: (event) => {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-    onKeyDown: (event) => {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'event' is declared but its value is never read.
-    onKeyUp: (event) => {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'position' is declared but its value is never read... Remove this comment to see the full error message
-    onPositioned: (position) => {},
-    // @ts-expect-error ts-migrate(6133) FIXME: 'position' is declared but its value is never read... Remove this comment to see the full error message
-    onPositionChanged: (position) => {},
-    renderTrigger: null,
-    children: null
+    shouldCloseOnEscape: true
   }
 
-  _handleMouseOver: (...args: any[]) => any | undefined
-  _handleMouseOut: (...args: any[]) => any | undefined
-
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  constructor(props) {
+  constructor(props: PopoverProps) {
     super(props)
 
     this.state = {
       placement: props.placement,
       offsetX: props.offsetX,
-      offsetY: props.offsetY
-    }
-
-    if (typeof props.isShowingContent === 'undefined') {
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'isShowingContent' does not exist on type... Remove this comment to see the full error message
-      this.state.isShowingContent = props.defaultIsShowingContent
+      offsetY: props.offsetY,
+      isShowingContent:
+        typeof props.isShowingContent === 'undefined'
+          ? props.defaultIsShowingContent
+          : undefined
     }
 
-    // @ts-expect-error ts-migrate(2339) FIXME: Property '_id' does not exist on type 'Popover'.
     this._id = this.props.id || uid('Popover')
     this._raf = []
 
-    this._handleMouseOver = handleMouseOverOut.bind(null, (event) => {
+    this._handleMouseOver = handleMouseOverOut.bind(null, (event: Event) => {
       this.show(event)
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'mouseOutTimeout' does not exist on type ... Remove this comment to see the full error message
-      clearTimeout(this.mouseOutTimeout)
+      clearTimeout(this.mouseOutTimeout!)
     })
     this._handleMouseOut = handleMouseOverOut.bind(null, (event) => {
       // this is needed bc the trigger mouseOut fires before tooltip mouseOver
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'mouseOutTimeout' does not exist on type ... Remove this comment to see the full error message
       this.mouseOutTimeout = setTimeout(() => {
         this.hide(event)
       }, 1)
     })
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'mouseOutTimeout' does not exist on type ... Remove this comment to see the full error message
     this.mouseOutTimeout = undefined
   }
 
+  _handleMouseOver: (event: MouseEvent) => void
+  _handleMouseOut: (event: MouseEvent) => void
```

Comment:
I've been playting with typing these properly.  Since there are on `View` they are actually `React.MouseEvent<any>`. If you type them this way you dont need the cast on line 536, also I'd type `handleMouseOverOut` as `event: { currentTarget: EventTarget | null, relatedTarget?: EventTarget | null}`

Reply:
React.MouseEventHandler did the trick, thanks!